OrbitSshQt/include/OrbitSshQt/SftpChannel.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_SSH_QT_SFTP_CHANNEL_H_
+#define ORBIT_SSH_QT_SFTP_CHANNEL_H_
+
+#include <QObject>
+#include <QPointer>
+#include <optional>
+#include <outcome.hpp>
+
+#include "OrbitSsh/Sftp.h"
+#include "OrbitSshQt/Base.h"
+#include "OrbitSshQt/Error.h"
+#include "OrbitSshQt/ScopedConnection.h"
+#include "OrbitSshQt/Session.h"
+
+namespace OrbitSshQt {
+namespace details {
+enum class SftpChannelState {
+  kInitial,
+  kNoChannel,
+  kStarted,
+  kChannelInitialized,
+  kShutdown,
+  kClosingChannel,
+  kDone,
+  kError
+};
+
+}
+class SftpChannel : public Base<SftpChannel, details::SftpChannelState> {
```

Comment:
add short description about this class, how it is supposed to be used and what it does.

Reply:
Done. Thanks!